docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/TC-PARAMETERS-5.md

```diff
@@ -0,0 +1,32 @@
+# Link storage with their URLs and not only with their IDs
+
+Test verifies that it's possible to limits number of S3 buckets mounted to an instance, defining buckets using their names instead of their IDs via CP_CAP_LIMIT_MOUNTS parameter.
+
+**Prerequisites**:
+- Admin user
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Create 2 object storages (`test_storage1`, `test_storage2`)
+3. Create pipeline (`test_pipeline1`)
+
+| Steps | Actions | Expected results |
+|:---:|---|---|
+| 1 | Open the **Library** page | |
+| 2 | Open the pipeline created at step 3 of the preparations | |
+| 3 | Click the pipeline draft version | |
+| 4 | Click the **CODE** tab | |
+| 5 | Click the `config.json` file | |
+| 6 | Click the **EDIT** button | |
+| 7 | Insert the following code into the `parameters` section: <br> `"CP_CAP_LIMIT_MOUNTS" : {` <br> `"value" : "<test_storage1_id>,<test_storage2_name>",` <br> `"type" : "string",` <br> `"required" : false` <br> `}` <br> where `<test_storage1_id>` - `test_storage1` id ,`<test_storage2_name>` - `test_storage1` name | |
```

Comment:
Fix to: "... where `<test_storage1_id>` - `test_storage1` id ,`<test_storage2_name>` - `test_storage2` name..."